Casks/j.rb

```diff
@@ -11,7 +11,10 @@
     app "j64-#{version}/#{a}.app"
   end
 
-  installer script: "j64-#{version}/updatejqt.sh"
+  installer script: {
+                      executable: "j64-#{version}/bin/jconsole",
+                      args: ['-js', "exit install'qtide'"]
```

Comment:
Does it work instead as:

```suggestion
                      args: ['-js', 'exit install"qtide"']
```

Reply:
Unfortuntely no. Strings in J can only use single quotes, since `"` is instead an operator.